### PR TITLE
rust automation

### PR DESCRIPTION
**Pull request by FuzzyNovaGoblin**

### CODE REVIEW

**Review by GigaRyno — APPROVED** [review]

File: src/matchers.rs [hunk 1]
```diff
@@ -85,42 +87,35 @@ pub enum MatcherType {
     Placeholder,
 }
 
+#[derive(Debug)]
+pub struct MatcherFuncArgs<'a> {
```
Comment (Redempt) [inline src/matchers.rs:91]: Can this be called MatcherContext or something?

**Review by boxbeam — CHANGES_REQUESTED** [review]

File: src/matchers.rs (hunk 1 above)
Comment (FuzzyNovaGoblin) [reply src/matchers.rs:91]: you don't like that the struct that is just supposed to be a wrapper for __arguments__ of the __matcher__ __functions__ is named as such?

Comment (Redempt) [reply src/matchers.rs:91]: Yes, I would prefer it be called MatcherContext, as it's the context being passed to matchers that they match with

MatcherFuncArgs just reads really weird

Comment (FuzzyNovaGoblin) [reply src/matchers.rs:91]: I thing `MatcherFuncArgs` reads normal and makes more sense but I don't actually care too much, I don't think `MatcherContext` is destructive

**Review by boxbeam — APPROVED** [review]

**Pull request merged**